src/language-js/print/statement.js

```diff
@@ -29,6 +34,12 @@ function printStatementSequence(path, options, print, property) {
     node[property].filter(({ type }) => type !== "EmptyStatement")
   );
 
+  const { originalText } = options;
+
+  if (isClassBody && isPreviousLineEmpty(originalText, node, locStart)) {
```

Comment:
another potential simplification:

```suggestion
  if (
    isClassBody &&
    isPreviousLineEmpty(options.originalText, node, locStart)
  ) {
```